auction-server/src/auction/repository/models.rs

```diff
@@ -127,13 +127,15 @@ pub trait ModelTrait<T: ChainTrait> {
         bid: &Bid<T>,
         auction: Option<Auction>,
     ) -> anyhow::Result<T::BidStatusType>;
+
+    fn convert_bid_status(status: &T::BidStatusType) -> BidStatus;
```

Comment:
Is the reason you don't implement `Into` here that the conversion is lossy?

Reply:
yup